content/en/opentelemetry/otlp_metrics.md

```diff
@@ -0,0 +1,284 @@
+  url: 'https://api.datadoghq.com/api/intake/otlp/v1/metrics',
+  temporalityPreference: AggregationTemporalityPreference.DELTA, // Ensure delta temporality
+  headers: {
+    'dd-api-key': process.env.DD_API_KEY,
+    'dd-otel-metric-config': '{resource_attributes_as_tags: true}',
+  },
+});
+```
+
+[100]: https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-proto
+
+{{% /tab %}}
+
+{{% tab "Java" %}}
+
+The Java exporter is [`OtlpHttpMetricExporter`][200]. To configure the exporter, use the following code snippet:
+
+```java
+import io.opentelemetry.exporter.otlp.http.metrics.OtlpHttpMetricExporter;
+
+OtlpHttpMetricExporter exporter = OtlpHttpMetricExporter.builder()
+    .setEndpoint("https://api.datadoghq.com/api/intake/otlp/v1/metrics")
+    .setAggregationTemporalitySelector(
+			AggregationTemporalitySelector.deltaPreferred()) // Ensure delta temporality
+    .addHeader("dd-api-key", System.getenv("DD_API_KEY"))
+    .addHeader("dd-otel-metric-config", "{resource_attributes_as_tags: true}")
+    .build();
+```
+
+[200]: https://javadoc.io/doc/io.opentelemetry/opentelemetry-exporter-otlp-http-metrics/
+
+{{% /tab %}}
+
+{{% tab "Go" %}}
+
+The Go exporter is [`otlpmetrichttp`][300]. To configure the exporter, use the following code snippet:
+
+```go
+import "go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp"
+
+metricExporter, err := otlpmetrichttp.New(
+	ctx,
+	otlpmetrichttp.WithEndpoint("api.datadoghq.com"),
+	otlpmetrichttp.WithURLPath("/api/intake/otlp/v1/metrics"),
+      otlpmetrichttp.WithTemporalitySelector(deltaSelector), // Ensure delta temporality
+	otlpmetrichttp.WithHeaders(
+		map[string]string{
+			"dd-api-key": os.Getenv("DD_API_KEY"),
+			"dd-otel-metric-config": "{resource_attributes_as_tags: true}",
+		}),
+)
+```
+
+[300]: https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp
+
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+The Python exporter is [`OTLPMetricExporter`][400]. To configure the exporter, use the following code snippet:
+
+```python
+from opentelemetry.exporter.otlp.proto.http.metric_exporter import OTLPMetricExporter
+
+exporter = OTLPMetricExporter(
+    endpoint="https://api.datadoghq.com/api/intake/otlp/v1/metrics",
+    preferred_temporality=deltaTemporality, # Ensure delta temporality
+    headers={
+        "dd-api-key": os.environ.get("DD_API_KEY"),
+        "dd-otel-metric-config": "{resource_attributes_as_tags: true}",
+    },
+)
+```
+
+[400]: https://pypi.org/project/opentelemetry-exporter-otlp-proto-http/
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### (Optional) Configure the metric translator
+
+Use the `dd-otel-metric-config` header to configure how metrics are translated and sent to Datadog. The JSON header contains the following fields:
+### Configure the metric translator
+
+Use the `dd-otel-metric-config` header to configure how metrics are translated and sent to Datadog. The JSON header contains the following fields:
```

Comment:
Some repeated text here, seems like you could delete 174-176?

Reply:
Good catch!